sky/execution.py

```diff
@@ -158,16 +159,32 @@ def _execute(
             cluster_name)
         cluster_exists = existing_handle is not None
 
+    stages = stages if stages is not None else list(Stage)
+
     backend = backend if backend is not None else backends.CloudVmRayBackend()
-    if not isinstance(backend, backends.CloudVmRayBackend
-                     ) and idle_minutes_to_autostop is not None:
+    if isinstance(backend, backends.CloudVmRayBackend):
+        if down and idle_minutes_to_autostop is None:
+            # Use autostop(down) to terminate the cluster after the task is
+            # done. Otherwise, the cluster will be immediately terminated when
```

Comment:
Wdym by "Otherwise, the cluster will be immediately terminated when            # the user detach from the execution."? Can be omitted?